03/components/Div.js

```diff
@@ -1,11 +1,15 @@
 // ./src/components/Div.js
-import React from 'react';
+import React, {useContext} from 'react';
+import { TextContext, ColorContext } from '../context';
 
 const Div = () => {
+    const redColor = useContext(ColorContext);
+    const text = useContext(TextContext);
+    const style = { color: redColor };
```

Comment:
Miało być obramowanie ;P
Dla mnie to bez znaczenia, ale jak robisz zadanie rekrutacyjne to możliwe, że jest ono sprawdzane przez algorytm i w tym momencie nie miałbyś już zaliczonego tego elementu ;)